proof/refine/ARM/Reply_R.thy

```diff
@@ -1160,27 +1160,33 @@ lemma valid_objs'_replyNexts_of_reply_at':
 
 (** sc_with_reply and sc_replies_relations : crossing information **)
 
+(* modified version of sc_replies_relation_prevs_list in StateRelation.thy;
+   updates only the abstract sc_relies; useful for the following few lemmas *)
 lemma sc_replies_relation_prevs_list':
   "\<lbrakk> sc_replies_relation s s';
-     kheap s scp = Some (kernel_object.SchedContext sc n)\<rbrakk>
-    \<Longrightarrow> heap_ls (replyPrevs_of s') (scReplies_of s' scp) (sc_replies sc)"
+     kheap s x = Some (kernel_object.SchedContext sc n)\<rbrakk>
+    \<Longrightarrow> heap_ls (replyPrevs_of s') (scReplies_of s' x) (sc_replies sc)"
```

Comment:
I will revert this back to `scp`a. This was once removed and then resurrected, but somehow didn’t quite make a full round-trip...